addons/l10n_latam_check/tests/test_own_checks.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo.addons.l10n_latam_check.tests.common import L10nLatamCheckTest
+from odoo.tests.common import Form, tagged
+from odoo import fields
+
+
+@tagged('post_install_l10n', 'post_install', '-at_install')
+class TestOwnChecks(L10nLatamCheckTest):
+
+    def test_01_pay_with_manual_check(self):
+        """ Create and post a manual check with deferred date """
+
+        with Form(self.env['account.payment'].with_context(default_payment_type='outbound')) as payment_form:
+            payment_form.partner_id = self.partner_a
+            payment_form.amount = 50
+            payment_form.journal_id = self.bank_journal
+            payment_form.payment_method_line_id = self.bank_journal._get_available_payment_method_lines(
+                'outbound').filtered(lambda x: x.code == 'check_printing')
+
+            payment_form.ref = 'Deferred check'
+            payment_form.l10n_latam_check_payment_date = fields.Date.add(fields.Date.today(), months=1)
+
+            # Manual check (deferred/electronic) has l10n_latam_manual_checks = True and do not auto compute the check
+            # number field
+            self.assertEqual(payment_form.l10n_latam_manual_checks, True)
+            self.assertEqual(payment_form.check_number, False)
+
+            payment_form.check_number = '00000001'
```

Comment:
l10n_latam_check_number then?  @zaoral Test is again failing...  (fields need to be visible if we want to set them with with Form...)

Reply:
@jco-odoo 
Indeed. should be l10n_latam_check_number. I have pushed the change. Please run the l10n sage again and let's wait for a new runbot build. Sorry about this, but I am not able to reproduce the errors in local and I am blind solving this errors :(

Reply:
@jco-odoo 
Please could you run the l10n stage tests